legate/tester/stages/_linux/omp.py

```diff
@@ -62,7 +62,7 @@ def shard_args(self, shard: Shard, config: Config) -> ArgList:
         if config.cpu_pin != "none":
             args += [
                 "--cpu-bind",
-                ",".join(str(x) for x in shard),
+                str(shard),
             ]
```

Comment:
```suggestion
            ]
        if config.ranks > 1:
            args += [
                "--ranks-per-node",
                str(config.ranks),
            ]
```

Doesn't look like `--ranks-per-node` is added in this path.